repose-aggregator/components/filters/versioning-filter/src/main/scala/org/openrepose/filters/versioning/VersioningFilter.scala

```diff
@@ -127,7 +123,7 @@ class VersioningFilter @Inject()(@Value(ReposeSpringProperties.NODE.CLUSTER_ID)
               httpResponse.setStatus(HttpServletResponse.SC_OK)
               httpResponse.addHeader("Content-Type", getPreferredMediaRange(wrappedRequest).getMimeType.getName)
 
-              mbcVersionedRequests.mark(targetOriginService.getMapping.getId)
+              markVersionedRequestMeter(targetOriginService.getMapping.getId)
```

Comment:
Not in this case. That is, there wasn't an all metric being reported by the `VersioningFilter`.

The `mbc` prefix here is calling out that this metric used to be a `MeterByCategory` metric. The `MeterByCategory` metric only ever help the 9/10th name. `MeterByCategorySum` is the metric that also created the all metric. That would be designated by a `mbcs` prefix in the old code.